src/tests/fixtures/projects/dummy-project.yaml

```diff
@@ -336,7 +337,8 @@ project-with-fastly-complex:
                     hostname: default.example.org
                 articles:
                     hostname: "{instance}-special.example.org"
-                    condition: req.url ~ "^/articles" 
+                    condition: req.url ~ "^/articles"
+                    shield: iad-va-us
```

Comment:
the meaning of `shielding` vs `shield` doesn't transpire. A way this is dealt with consistently for other options is to make the positive value of `shielding` a dictionary containing an optional key `pop`:

```
shielding:
    pop: iad-va-us
# or, if you want the default
shielding: True
# (as an empty dictionary would still be Falsy)
```

It works much better when there are already other values, but it leave space for them to be inserted if needed. Any other setting comes to mind, that for example would customize a VCL snippet or influence something else?

It does take away support for defaults shielding configuration on projects with `shielding` turned off, but those can be left as:
```
#shielding:
#    pop: iad-va-us
```

Reply:
Could rename `shielding` `shield` and trigger a failure if both `backends` and `shield` are set (as they're incompatible)?

Don't know if we would ever need any other values, but don't mind making it a dictionary.